### DISCUSSION TITLE
Amazon "Buyer's option".

### DISCUSSION THREAD

**Discussion by xmpierce1** [Prince25/StockAlertBot#80] [Requests]
With some Amazon product pages the other options page automatically comes up with a lower price from Amazon itself. It's kind of confusing so I pasted an example. How can StockAlert handle this case? 
![image](https://user-images.githubusercontent.com/74942520/114277945-7133e700-99fb-11eb-88c7-19b6f8918be5.png)

**Comment by Prince25** [comment]
Hello. Are you suggesting a feature to get price from a particular merchant? If so, this is already added: `AMAZON_MERCHANT_ID` in `config.json`.
